src/plugins/assume-nonnull.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import AlgebraicType = require('../algebraic-type');
+import Code = require('../code');
+import Error = require('../error');
+import FileWriter = require('../file-writer');
+import Maybe = require('../maybe');
+import ObjC = require('../objc');
+import ValueObject = require('../value-object');
+
+function assumeNonnullFileTransformation(request:FileWriter.Request):FileWriter.Request {
```

Comment:
@ColinCampbell Yeah what you're suggesting seems like a better approach. It should also make writing unit tests better compared to what I have right now.